common/shared.c

```diff
@@ -65,7 +65,7 @@ size_t buf_size, tx_size, rx_size;
 int rx_fd = -1, tx_fd = -1;
 char default_port[8] = "9228";
 
-char test_name[10] = "custom";
+char test_name[50] = "custom";
```

Comment:
gurgle, I think we need to shorten this, even if it means editing the fi_tostr() output.  We can at least drop the "FI_" that's repeated twice and predictable.  And given that the atomic operation is listed, I think we can drop the 'fetch_' from the name (likewise for 'base_').

That would at least get us to "LONG_DOUBLE_COMPLEX_ATOMIC_READ_lat", which is still ugly long, but slightly better.  Anything else basically results in a custom fi_tostr()...